portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.data;
+
+import java.time.Duration;
+
+/** Models a task. It can be used for both creation and import flow. */
+public final class Task {
+
+  private final String name;
+  private final String description;
+  private final Duration durationMinute;
```

Comment:
Now that we're using the Duration class, we can omit the unit.

durationMinute => duration

Reply:
Done.